src/server/create/microservice/controllers/detail.js

```diff
@@ -54,14 +54,17 @@ const microserviceDetailController = {
     if (!validationResult.error) {
       const usersTeams = await getUsersTeams(request)
       const team = usersTeams.find((team) => team.teamId === teamId)
-      const serviceTypeDetail = serviceTypes.find(
-        (serviceType) => serviceType.value === serviceTypeTemplate
+      const serviceTemplateDetail = serviceTemplates.find(
+        (serviceTemplate) =>
+          serviceTemplate.repositoryName === serviceTypeTemplate
```

Comment:
nit: part of me is wondering if the change of name should also be in the input name. So `payload.serviceTypeTemplate` becomes `payload.repositoryName` ?

Not the end of the world hence marked as a nit, but there is a swap of names in this controller and it might be clearer if it was the same name all the way through?

Reply:
its not the repositoryName, repositoryName is name of the service being created. serviceTypeTemplate is name of the template repo. this got me too.

Reply:
wait so is the above comparison correct?